common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -117,6 +117,14 @@ class ProctoringFields(object):
     )
 
     @property
+    def is_timed_exam(self):
+        """
+        Alias the permutation of above fields that corresponds to un-proctored timed exams
+        to the more clearly-named is_timed_exam
+        """
+        return not self.is_proctored_enabled and self.is_time_limited
```

Comment:
Nit: Can you keep the `is_proctored_exam` and `is_practice_exam` properties next to each other since they are related?